Sources/Extensions/UIKit/UITextFieldExtensions.swift

```diff
@@ -123,6 +125,24 @@ public extension UITextField {
 			iconView.tintColor = newValue
 		}
 	}
+    
+    /// SwifterSwift: Next text field to become first responder
+    ///
+    ///     textField.nextField = someOtherTextField
+    ///     textField.nextField?.becomeFirstResponder() -> 'someOtherTextField` is now first responder
+    ///
+    /// https://stackoverflow.com/a/27030181
+    @IBOutlet public var nextField: UITextField? {
+        
```

Comment:
I think this should probably be marked `weak`. However, it may not be a good fit for the library. I'm not fond of the global constant and I can't help but feel there is a Swifter way of doing this.

Reply:
Agreed on 'weak'. We can remove the global constant and just put '0' in that field. I'm not sure of any other way to make a property available to IB. I don't think @IBDesignable is applicable to this. So, perhaps I just remove this change.